src/tests/api/Filter.test.js

```diff
@@ -0,0 +1,136 @@
+import { applyFilter } from 'src/api';
+import { filters } from './data-mocks/filters';
+
+const makePossibleFeature = (value) => [
+  {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [0, 0]
+    },
+    properties: {
+      column1: value
+    }
+  },
+  {
+    column1: value
+  }
+];
+
+describe('Filters', () => {
+  test('should return 1 if no filters present', () => {
+    const params = { filters: {}, type: 'number' };
+    expect(applyFilter(params)()).toBe(1);
+  });
+
+  test('should return true if no filters present', () => {
+    const params = { filters: {}, type: 'boolean' };
+    expect(applyFilter(params)()).toBe(true);
+  });
+
+  describe('feature passes filter - boolean type', () => {
+    const params = { filters, type: 'boolean' };
+
+    describe('should return false if feature column value is falsy', () => {
+      const testCases = [0, null, undefined, false, ''];
+      for (const tc of testCases) {
+        test(String(tc), () => {
+          const [featureWithProperties, rawFeature] = makePossibleFeature(tc);
+          const withProps = applyFilter(params)(featureWithProperties);
+          expect(withProps).toBe(false);
+          const noProps = applyFilter(params)(rawFeature);
```

Comment:
I try not to mix several **expect** in the same test, as they tend to be more complex... In this case, I have no clear idea about the 'rawFeature'... but I would probably suggest extracting that to another test, even if that requires a bit of duplication